### PR TITLE
Account for extraneous weight and height in UG

### LINKED ISSUE #266: [PE-D][Tester C] `Weight` command: Fix behaviour when multiple `[WEIGHT]` input is given
Input: `weight 1 10 20 30 40 50`
Expected: Error message specifying that `weight` command can only take in 1 [WEIGHT] argument.
Actual: Command succeeds, only taking the first weight argument.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: shaunlxw/ped#9

### PR DESCRIPTION
**Pull request by ruijietay**
Resolves #266.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -143,6 +143,9 @@ Adds a client to the FitBook.
 
 Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGHT] [nt/NOTE] [t/TAG]…​`
 
+
+* HEIGHT and WEIGHT each takes in only one value. (e.g. `add n/Tom p/123 w/85 65` or `add n/Tom p/123 h/180 175` is invalid).
```
Comment (dillontkh) [inline docs/UserGuide.md:147]:
```suggestion
* HEIGHT and WEIGHT only take in one value each. (e.g. `add n/Tom p/123 w/85 65` or `add n/Tom p/123 h/180 175` is invalid).
```

**Review by dillontkh — COMMENTED** [review]
I think this should be mentioned in the parameter format section to avoid duplication of information.

File: docs/UserGuide.md (hunk 1 above)
Comment (jjchee77) [reply docs/UserGuide.md:147]: If its not too much, separating them onto their own lines will be better.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -251,6 +255,7 @@ Format: `weight INDEX [WEIGHT] `
 
 * Adds a weight value to a client specified by `INDEX`. The index refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​
 * Weight values specified in this command will be added as a new weight value to the specified client. To edit the latest weight of the client, use the [`edit`](#editing-a-client--edit) command.
+* If more than one value is entered, only the first value will be parsed. Extraneous parameters after the first value will be ignored.
```
Comment (jjchee77) [inline docs/UserGuide.md:258]: Might want to provide an example here for clarity.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -172,6 +175,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGH
 * Existing values will be updated to the input values.
 * When no value is specified after a prefix, the value of that prefix will be removed from the client. (e.g. `edit 1 nt/` removes the note from the client at index 1).
     * Since WEIGHT can store multiple historical values, specifying an empty weight prefix removes the latest weight value from the client. Otherwise, the latest weight value will be replaced by the specified weight in this command.
+* HEIGHT and WEIGHT each takes in only one value. (e.g. `edit 1 h/180 175` or `edit 1 w/85 65` is invalid).
```
Comment (bgopi23) [inline docs/UserGuide.md:178]:
```suggestion
* HEIGHT and WEIGHT only take in only one value each. (e.g. `edit 1 h/180 175` or `edit 1 w/85 65` is invalid).
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -172,6 +175,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGH
...
 * When editing tags, the existing tags of the client will be removed i.e adding of tags is not cumulative.
```
Comment (bgopi23) [inline docs/UserGuide.md:179]:
```suggestion
* When editing tags, the existing tags of the client will be removed i.e. adding of tags is not cumulative.
```

File: docs/UserGuide.md (hunk 2 above)
Comment (bgopi23) [reply docs/UserGuide.md:258]: e.g. (weight 1 90 80 90 100)

**Review by RyanNgWH — APPROVED** [review]
LGTM!

**Review by bgopi23 — APPROVED** [review]
LGTM!

**Review by dillontkh — APPROVED** [review]
LGTM!

**Pull request merged**